### PR TITLE
[linstor] Added section about data nodes nodeSelector in docs

### PR DESCRIPTION
**Pull request by duckhawk**
## Description

Added section about data nodes nodeSelector in docs

## Why do we need it, and what problem does it solve?

We have no mentions about data nodes nodeSelector in documentation

## What is the expected result?

Actual documentation

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: linstor
type: chore
summary: Added section about data nodes nodeSelector in docs.
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Review by z9r5 — APPROVED** [review]

File: modules/041-linstor/docs/USAGE_RU.md [hunk 1]
```diff
@@ -1,11 +1,100 @@
 ---
 title: "Модуль linstor: примеры конфигурации"
+description: Примеры конфигурации модуля linstor Deckhouse. Конфигурация хранилища LINSTOR. Настройка data-locality - размещение приложения возле данных. Storage-based fencing. Перенос приложения при проблемах с узлом.
```
Comment (apolovov) [inline modules/041-linstor/docs/USAGE_RU.md:3]:
```suggestion
description: Примеры конфигурации модуля linstor. Настройка data-locality — размещение приложения возле данных. Storage-based fencing. Перенос приложения при проблемах с узлом.
```

File: modules/041-linstor/docs/USAGE_RU.md [hunk 2]
```diff
@@ -1,11 +1,100 @@
...
 ---
 
-Модуль linstor поддерживает два метода конфигурации:
+{% alert %}
+Работоспособность модуля гарантируется только для стоковых ядер, поставляемых вместе с дистрибутивами перечисленными в [списке поддерживаемых ОС](../../supported_versions.html#linux).
```
Comment (apolovov) [inline modules/041-linstor/docs/USAGE_RU.md:7]:
```suggestion
Работоспособность модуля гарантируется только для стоковых ядер, поставляемых вместе с дистрибутивами, перечисленными в [списке поддерживаемых ОС](../../supported_versions.html#linux).
```

File: modules/041-linstor/docs/USAGE_RU.md [hunk 3]
```diff
@@ -1,11 +1,100 @@
...
+Работоспособность модуля на других ядрах возможна, но не гарантируется.
+{% endalert %}
 
-- Автоматическая конфигурация на основе тегов LVM. Для этого обратитесь к странице [Настройки](configuration.html).
-- Ручная конфигурация используя LINSTOR CLI. Для этого обратитесь к [Расширенной конфигурации](advanced_usage.html).
+После включения модуля кластер автоматически настраивается на использование LINSTOR и остается только сконфигурировать хранилище.
```
Comment (apolovov) [inline modules/041-linstor/docs/USAGE_RU.md:11]:
```suggestion
После включения модуля, кластер автоматически настраивается на использование LINSTOR и остается только сконфигурировать хранилище.
```

File: modules/041-linstor/docs/USAGE_RU.md [hunk 4]
```diff
@@ -1,11 +1,100 @@
...
+
+Отдельные функции модуля могут потребовать указания мастер-пароля.
```
Comment (apolovov) [inline modules/041-linstor/docs/USAGE_RU.md:13]: Уточнить пример, где нужен мастер-пароль.

File: modules/041-linstor/docs/USAGE_RU.md [hunk 5]
```diff
@@ -1,11 +1,100 @@
...
+Работоспособность модуля гарантируется только для стоковых ядер, поставляемых вместе с дистрибутивами перечисленными в [списке поддерживаемых ОС](../../supported_versions.html#linux).
+Работоспособность модуля на других ядрах возможна, но не гарантируется.
+{% endalert %}
 
-- Автоматическая конфигурация на основе тегов LVM. Для этого обратитесь к странице [Настройки](configuration.html).
-- Ручная конфигурация используя LINSTOR CLI. Для этого обратитесь к [Расширенной конфигурации](advanced_usage.html).
+После включения модуля кластер автоматически настраивается на использование LINSTOR и остается только сконфигурировать хранилище.
+
+Отдельные функции модуля могут потребовать указания мастер-пароля.  
+Для того чтобы задать мастер-пароль, создайте Secret в пространстве имен `d8-system`:
```
Comment (apolovov) [inline modules/041-linstor/docs/USAGE_RU.md:14]:
```suggestion
Для того чтобы задать мастер-пароль, создайте Secret в пространстве имён `d8-system`:
```

File: modules/041-linstor/docs/USAGE_RU.md [hunk 6]
```diff
@@ -1,11 +1,100 @@
...
+metadata:
+  name: linstor-passphrase
+  namespace: d8-system
+immutable: true
+stringData:
+  MASTER_PASSPHRASE: *!пароль* # Мастер-пароль для LINSTOR
+```
+
+{% alert level="warning" %}
+Подойдите ответственно к выбору мастер-пароля для LINSTOR. При его потере зашифрованные данные окажутся недоступными.
```
Comment (apolovov) [inline modules/041-linstor/docs/USAGE_RU.md:28]:
```suggestion
Подойдите ответственно к выбору мастер-пароля для LINSTOR. При его потере, зашифрованные данные окажутся недоступными.
```
По-моему, запятая нужна.

**Review by apolovov — CHANGES_REQUESTED** [review]

File: modules/041-linstor/docs/USAGE_RU.md (hunk 3 above)
Comment (z9r5) [reply modules/041-linstor/docs/USAGE_RU.md:11]: IMHO запятая не нужна.

Comment (z9r5) [reply modules/041-linstor/docs/USAGE_RU.md:11]: @kirkonru your turn :)

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

File: modules/041-linstor/docs/CONFIGURATION_RU.md [hunk 7]
```diff
@@ -1,95 +1,7 @@
 ---
 title: "Модуль linstor: настройки"
 force_searchable: true
+description: Параметры настройки модуля linstor Deckhouse
```
Comment (z9r5) [inline modules/041-linstor/docs/CONFIGURATION_RU.md:4]:
```suggestion
description: Параметры настройки модуля linstor Deckhouse.
```

**Review by apolovov — APPROVED** [review]

**Review by AleksZimin — APPROVED** [review]

**Review by nabokihms — APPROVED** [review]

**Pull request merged**